packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -235,27 +235,49 @@ const helpers = {
     return helpers.negotiateOptionsForDefaultCode(httpOperation, desiredOptions);
   },
 
+  find400Or422(httpResponses: IHttpOperationResponse[], logger: Logger) {
+    let result = findResponseByStatusCode(httpResponses, '422');
+    if (!result) {
+      logger.trace('Unable to find a 422 response definition');
+
+      result = findResponseByStatusCode(httpResponses, '400');
+      if (!result) {
+        logger.trace('Unable to find a 400 response definition.');
+        const response = createResponseFromDefault(httpResponses, '422');
+        if (response) logger.success(`Created a ${response.code} from a default response`);
+        return response;
+      }
+    }
+
+    logger.success(`Found response ${result.code}. I'll try with it.`);
+    return result;
+  },
+
+  findByStatusCode(statusCode: string, httpResponses: IHttpOperationResponse[]) {
```

Comment:
It looks like this method is just aliasing `findResponseByStatusCode` and sapping the order of arguments? Is this needed?

Reply:
ha! thanks :) not needed indeed :+1: